src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -201,8 +204,8 @@ const ChatItemMyTextV2 = ({
   }, []);
 
   const renderAvatar = React.useCallback(() => {
-    if (isContinuous) return <View style={styles.avatar} />;
-    return <View style={styles.ml8}>{avatar}</View>;
+    if (isContinuous) return <View style={[styles.avatar, styles.marginContinuous]} />;
+    return <View style={styles.marginNonContinuous}>{avatar}</View>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The renderAvatar function has been updated to use the new margin styles based on the `isContinuous` flag. This seems like a good way to handle different rendering conditions for the avatar.


<!-- This is an auto-generated comment by OSS CodeRabbit -->